src/DataFixtures/ORM/LoadTerritorialCouncilMembershipLogData.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\DataFixtures\ORM;
+
+use App\Entity\TerritorialCouncil\TerritorialCouncilMembershipLog;
+use App\Entity\TerritorialCouncil\TerritorialCouncilQualityEnum;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Common\Persistence\ObjectManager;
+
+class LoadTerritorialCouncilMembershipLogData extends Fixture
+{
+    public function load(ObjectManager $manager)
+    {
+        $membershipLog1 = new TerritorialCouncilMembershipLog(
+            TerritorialCouncilMembershipLog::TYPE_WARNING,
+            'Plusieurs conseils territorials ont été trouvés pour cette qualité',
+            $this->getReference('adherent-7'),
+            TerritorialCouncilQualityEnum::COMMITTEE_SUPERVISOR,
+            null,
+            [],
+            ['Seine-et-Marne (77), Essonne (91)']
```

Comment:
should this be :
```suggestion
            ['Seine-et-Marne (77)', 'Essonne (91)']
```
?